fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/FedoraOCFLMapping.java

```diff
@@ -17,11 +17,14 @@
  */
 package org.fcrepo.persistence.ocfl.impl;
 
+import org.springframework.stereotype.Component;
+
 /**
  * A mapping that links the parent fedora resource to its corresponding OCFL object.
  *
  * @author dbernstein
  */
+@Component
```

Comment:
This @Component doesn't belong here.  It should go on the implementation of FedoraToOCFLObjectIndex when it exists.